docker/docker-compose.yml

```diff
@@ -89,9 +88,15 @@ services:
     tty: true
     command: bundle exec rails server -b 0.0.0.0
     labels:
-      - "traefik.http.routers.op.rule=Host(`open-path-cas.127.0.0.1.nip.io`)"
-      - "traefik.http.routers.op.tls=true"
-      - "traefik.http.services.op.loadbalancer.server.port=3000"
+      - traefik.enable=true
+      - traefik.http.routers.cas.entrypoints=web
+      - traefik.http.routers.cas.rule=Host(`boston-cas.dev.test`)
+      - traefik.http.services.cas_https.loadbalancer.server.port=3000
+      - traefik.http.routers.cas_https.rule=Host(`boston-cas.dev.test`)
```

Comment:
@dtgreiner I _think_ we want these to reference the `FQDN` environment variable as they do in the warehouse since people have different names for the app (presumably).  Can you update that?

Reply:
You're right and thanks for catching that! I'm including the TRAEFIK_ENABLED environment variable as well

Reply:
Thanks!  I only caught it because I couldn't get it to work locally, so I knew to check for it.  Were you able to confirm those changes worked? (after docker-compose restart)

Reply:
Yes, I am currently running CAS locally with this docker-config file. As long as DIRENV has been configured in the CAS environment, it should be good to go.

Reply:
Ok, seems to work for me now, so I must have had something wrong.  Thanks!  Merging.